Build fails on Arch Linux

##### LibreSprite version
```
master branch
```

##### Describe the bug
The build fails with the following error:
<details>
<summary>Error</summary>

```
[606/717] Building CXX object src/app/CMakeFiles/app-lib.dir/tools/active_tool.cpp.o
FAILED: src/app/CMakeFiles/app-lib.dir/tools/active_tool.cpp.o 
/usr/bin/c++ -DHAVE_CONFIG_H -DNDEBUG -DSCRIPT_ENGINE_LUA=1 -DUSE_SDL2_BACKEND -I/usr/include/pixman-1 -I/usr/include/freetype2 -I/home/marco/Workspace/LibreSprite/third_party/simpleini -I/usr/include/SDL2 -I/home/marco/Workspace/LibreSprite/src/. -I/home/marco/Workspace/LibreSprite/src/.. -I/home/marco/Workspace/LibreSprite/src/../third_party -I/home/marco/Workspace/LibreSprite/src/observable -I/home/marco/Workspace/LibreSprite/build/src/base -I/home/marco/Workspace/LibreSprite/build/src/app -std=c++14 -Wall -Wno-switch -std=gnu++11 -std=c++14 -O2 -g -DNDEBUG -D_REENTRANT -MD -MT src/app/CMakeFiles/app-lib.dir/tools/active_tool.cpp.o -MF src/app/CMakeFiles/app-lib.dir/tools/active_tool.cpp.o.d -o src/app/CMakeFiles/app-lib.dir/tools/active_tool.cpp.o -c /home/marco/Workspace/LibreSprite/src/app/tools/active_tool.cpp
In file included from /home/marco/Workspace/LibreSprite/src/app/tools/active_tool.cpp:12:
/home/marco/Workspace/LibreSprite/src/./app/tools/active_tool.h:28:8: error: ‘shared_ptr’ in namespace ‘std’ does not name a template type
   28 |   std::shared_ptr<Ink> activeInk() const;
      |        ^~~~~~~~~~
/home/marco/Workspace/LibreSprite/src/./app/tools/active_tool.h:11:1: note: ‘std::shared_ptr’ is defined in header ‘<memory>’; did you forget to ‘#include <memory>’?
   10 | #include "base/observable.h"
  +++ |+#include <memory>
   11 | 
/home/marco/Workspace/LibreSprite/src/./app/tools/active_tool.h:58:8: error: ‘shared_ptr’ in namespace ‘std’ does not name a template type
   58 |   std::shared_ptr<Ink> m_rightClickInk;
      |        ^~~~~~~~~~
/home/marco/Workspace/LibreSprite/src/./app/tools/active_tool.h:58:3: note: ‘std::shared_ptr’ is defined in header ‘<memory>’; did you forget to ‘#include <memory>’?
   58 |   std::shared_ptr<Ink> m_rightClickInk;
      |   ^~~
/home/marco/Workspace/LibreSprite/src/app/tools/active_tool.cpp: In constructor ‘app::tools::ActiveToolManager::ActiveToolManager(app::tools::ToolBox*)’:
/home/marco/Workspace/LibreSprite/src/app/tools/active_tool.cpp:49:5: error: class ‘app::tools::ActiveToolManager’ does not have any field named ‘m_rightClickInk’
   49 |   , m_rightClickInk(nullptr)
      |     ^~~~~~~~~~~~~~~
/home/marco/Workspace/LibreSprite/src/app/tools/active_tool.cpp: At global scope:
/home/marco/Workspace/LibreSprite/src/app/tools/active_tool.cpp:71:22: error: no declaration matches ‘std::shared_ptr<app::tools::Ink> app::tools::ActiveToolManager::activeInk() const’
   71 | std::shared_ptr<Ink> ActiveToolManager::activeInk() const
      |                      ^~~~~~~~~~~~~~~~~
/home/marco/Workspace/LibreSprite/src/app/tools/active_tool.cpp:71:22: note: no functions named ‘std::shared_ptr<app::tools::Ink> app::tools::ActiveToolManager::activeInk() const’
In file included from /home/marco/Workspace/LibreSprite/src/app/tools/active_tool.cpp:12:
/home/marco/Workspace/LibreSprite/src/./app/tools/active_tool.h:23:7: note: ‘class app::tools::ActiveToolManager’ defined here
   23 | class ActiveToolManager : public base::Observable<ActiveToolObserver> {
      |       ^~~~~~~~~~~~~~~~~
/home/marco/Workspace/LibreSprite/src/app/tools/active_tool.cpp: In member function ‘void app::tools::ActiveToolManager::pressButton(const app::tools::Pointer&)’:
/home/marco/Workspace/LibreSprite/src/app/tools/active_tool.cpp:189:3: error: ‘m_rightClickInk’ was not declared in this scope; did you mean ‘m_rightClick’?
  189 |   m_rightClickInk = ink;
      |   ^~~~~~~~~~~~~~~
      |   m_rightClick
/home/marco/Workspace/LibreSprite/src/app/tools/active_tool.cpp: In member function ‘void app::tools::ActiveToolManager::releaseButtons()’:
/home/marco/Workspace/LibreSprite/src/app/tools/active_tool.cpp:198:3: error: ‘m_rightClickInk’ was not declared in this scope; did you mean ‘m_rightClick’?
  198 |   m_rightClickInk = nullptr;
      |   ^~~~~~~~~~~~~~~
      |   m_rightClick
[615/717] Building CXX object src/app/CMakeFiles/app-lib.dir/tools/tool_box.cpp.o
In file included from /home/marco/Workspace/LibreSprite/src/./app/tools/inks.h:8,
                 from /home/marco/Workspace/LibreSprite/src/app/tools/tool_box.cpp:36:
/home/marco/Workspace/LibreSprite/src/./app/tools/ink_processing.h: In member function ‘void app::tools::{anonymous}::LockAlphaInkProcessing<doc::IndexedTraits>::processPixel(int, int)’:
/home/marco/Workspace/LibreSprite/src/./app/tools/ink_processing.h:184:11: warning: comparison of integer expressions of different signedness: ‘doc::color_t’ {aka ‘unsigned int’} and ‘const int’ [-Wsign-compare]
  184 |     if (c == m_maskIndex)
      |         ~~^~~~~~~~~~~~~~
/home/marco/Workspace/LibreSprite/src/./app/tools/ink_processing.h: In member function ‘void app::tools::{anonymous}::TransparentInkProcessing<doc::IndexedTraits>::processPixel(int, int)’:
/home/marco/Workspace/LibreSprite/src/./app/tools/ink_processing.h:249:11: warning: comparison of integer expressions of different signedness: ‘doc::color_t’ {aka ‘unsigned int’} and ‘const int’ [-Wsign-compare]
  249 |     if (c == m_maskIndex)
      |         ~~^~~~~~~~~~~~~~
/home/marco/Workspace/LibreSprite/src/./app/tools/ink_processing.h: In constructor ‘app::tools::{anonymous}::MergeInkProcessing<doc::IndexedTraits>::MergeInkProcessing(app::tools::ToolLoop*)’:
/home/marco/Workspace/LibreSprite/src/./app/tools/ink_processing.h:308:37: warning: comparison of integer expressions of different signedness: ‘doc::color_t’ {aka ‘unsigned int’} and ‘const int’ [-Wsign-compare]
  308 |     m_color(loop->getPrimaryColor() == m_maskIndex ?
      |             ~~~~~~~~~~~~~~~~~~~~~~~~^~~~~~~~~~~~~~
/home/marco/Workspace/LibreSprite/src/./app/tools/ink_processing.h: In member function ‘void app::tools::{anonymous}::MergeInkProcessing<doc::IndexedTraits>::processPixel(int, int)’:
/home/marco/Workspace/LibreSprite/src/./app/tools/ink_processing.h:315:11: warning: comparison of integer expressions of different signedness: ‘doc::color_t’ {aka ‘unsigned int’} and ‘const int’ [-Wsign-compare]
  315 |     if (c == m_maskIndex)
      |         ~~^~~~~~~~~~~~~~
ninja: build stopped: subcommand failed.
```
</details>

##### To Reproduce
Build the project

##### Expected behaviour
Project should build

##### OS/Hardware
- Operative System: Arch Linux
- CPU: i7 7700HQ

--- a/src/app/tools/active_tool.h
+++ b/src/app/tools/active_tool.h
@@ -9,6 +9,8 @@
 
 #include "base/observable.h"
 
+#include <memory>
+
 namespace app {
 namespace tools {
 

